RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHBHE_cfi.py

```diff
@@ -31,3 +42,64 @@
 
 )
 
+cuts2017 = particleFlowRecHitHBHE.producers[0].qualityTests[0].cuts
+
+cuts2018 = cms.VPSet(
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4),
+        threshold = cms.vdouble(0.8, 0.8, 0.8, 0.8),
+        detectorEnum = cms.int32(1)
+        ),
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4, 5, 6, 7),
+        threshold = cms.vdouble(0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2),
+        detectorEnum = cms.int32(2)
+        )
+    )
+
+cuts2019 = cms.VPSet(
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4),
+        threshold = cms.vdouble(0.1, 0.2, 0.3, 0.3),
+        detectorEnum = cms.int32(1)
+        ),
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4, 5, 6, 7),
+        threshold = cms.vdouble(0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2),
+        detectorEnum = cms.int32(2)
+        )
+    )
+
+cutsPhase2 = cms.VPSet(
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4),
+        threshold = cms.vdouble(0.8, 0.8, 0.8, 0.8),
```

Comment:
why is phase-2 cut tighter than 2019?

Reply:
I think the idea is that the dark current noise increases as time goes on (our raddam model for SiPM), so the tigher cut would make sense than 2019. I don't think we really optimized this cut for phase2, but this value has been used for various phase2 studies for a long time. So, it would be good to stay with these numbers for now.